docs/book/src/troubleshooting.md

```diff
@@ -53,4 +53,13 @@ Secrets Store CSI Driver is deployed as a *DaemonSet*. The above error indicates
   NAME                       ATTACHREQUIRED   PODINFOONMOUNT   MODES       AGE
   secrets-store.csi.k8s.io   false            true             Ephemeral   110m
   ```
-  
\ No newline at end of file
+
+### Mount fails with `grpc: received message larger than max`
+
+The files pulled in by the `SecretProviderClass` are larger than 4MiB you may observe `FailedMount` warnings with a
```

Comment:
This sentence doesn't seem to be complete. 

If the files pulled in by the `SecretProviderClass` are larger than 4MiB you may observe `FailedMount` warnings with a message that include `grpc: received message larger than max`.

Reply:
😢 fixed